src/os/bluestore/KernelDevice.h

```diff
@@ -34,7 +34,8 @@ class KernelDevice : public BlockDevice {
   interval_set<uint64_t> debug_inflight;
 
   Mutex flush_lock;
```

Comment:
There are 3 flush_lock-s around BlueStore - one at Onode, another one at KernelDevice and third one in SharedDriverData at NVMEDevice.cc. What you're saying is valid for Onode one. while I was referring for KernelDevice member. IMO it's not used...